website/general_issues.qmd

```diff
@@ -6,4 +6,63 @@ format:
     css: style.css
 ---
 
-# tbd
+# Structure of CRAN-Submissions
+
+<!-- I would add a section about the path of packages when they are submitted to CRAN.
+Where do they end up and who will look at them etc... -->
+
+# Fields the DESCRIPTION File
+
+<!-- I suggest writing a short overview of the most common and important fields of the DESCRIPTION -->
+
+# Structuring of Examples 
+
+As part their documentation most functions have example code in their .Rd-files marked by the `\examples{}` tag. 
+Examples serve two major purposes for packages on CRAN.
+
+1. Show users how to implement the function correctly and what to expect as outcome.
+2. Allow the CRAN team to perform test on the code.
+
+For several reasons, some examples cannot be run as tests. Therefore, CRAN has several wrappers to structure them. The following list gives a short overview of the most common ones and when to use them.
```

Comment:
Maybe also mention the vignettes that are longer than examples but are also tested/checked.